igsr_archive/api.py

```diff
@@ -71,55 +71,41 @@ def retrieve_object(self, fireOid=None, firePath=None, outfile=None):
         ----------
         fireOid : str, optional
```

Comment:
Thank you for the fix, @olaaustine. The code looks good and works as expected. Do we still want to take fireOid as an input to this function as it is not required anymore?

Reply:
Hi @likhitha-surapaneni,
Thank you for reviewing the PR.
Yes i thought it best to leave it for now and added the error message and exit if it is used and also updated the documentation in the code  before it is removed finally. 
If you think a final removal is a better option, I can do that. 
Thank you. 
Ola.